src/app/features/charts/components/score-chart/score-chart.component.spec.ts

```diff
@@ -39,7 +39,7 @@ describe('Component: Chart - Score', () => {
   });
 
   it('should display message if not data', () => {
-    expect(component.score).toEqual(0);
+    expect(component.score).toEqual(null);
     expect(element.querySelector('.chart-no-data')).toBeTruthy();
   });
```

Comment:
You could consider also verifying that a 0 score is treated as an actual value, if you want to capture both sides of the edge case. Might be kind of awkward with how this test is set up.

Reply:
Great suggestion.  I added zero and valid value cases.